src/main/java/org/digma/intellij/plugin/jaegerui/JaegerUIResourceHandler.java

```diff
@@ -0,0 +1,130 @@
+package org.digma.intellij.plugin.jaegerui;
```

Comment:
is this file new? what is its responsibility?

Reply:
To load the correct resources for jcef browser jaeger ui